www/index.md

```diff
@@ -65,3 +65,9 @@ Nächstes Treffen
 		<a href="pizza.html">Möchtest Du Pizza mitbestellen?</a>
 	{% endif %}
 </p>
+
+
+Aktuelle Veranstaltungsreihe
```

Comment:
This would be subject to internal discussion if and how we link this on the main page, as this is not the only event the NoName organizes. Therefore please no changes on the main page in this PR. Could be discussed in a separate one.